### PR TITLE
Reconfigure RDWatch CLI to get BBoxes directly

### PR DESCRIPTION
**Pull request by BryonLewis**
This PR will reconfigure the RDWatch CLI to get bboxes directly instead of trying to convert to tiles and stitching the area together.

Using the argument `--tile` will use the old behavior of returning a very large image using the tile method of getting pixels.  Without that argument, it will default to using the bounding box instead.

### CODE REVIEW

File: cli/src/rdwatch_cli/api/image.py
Comment (mvandenburgh) [inline cli/src/rdwatch_cli/api/image.py:20]: imo `ObsImage` is too unclear for someone not familiar with the naming terminology for our internal data models.
```suggestion
class ObservationImage:
```

File: django/src/rdwatch/views/tile.py
Comment (mvandenburgh) [inline django/src/rdwatch/views/tile.py:258]: I think at the minimum it would be good to factor out the common code between this endpoint and the original `z/x/y/` endpoint into a common function, if possible. I'm less sure about this, but maybe a more ideal solution would be to remove the `z/x/y/` endpoint, and have a single endpoint that takes as query parameters {`z`, `x`, `y`} OR {`bbox` }. And for now we can make the `z/x/y` redirect to the new one until the frontend is updated accordingly. But I'm conflicted on whether that is better, since the `z/x/y/` url pattern is common in GIS systems I believe. Any thoughts on this? In general, I fear that we're adding too many similar endpoints for satellite images and the API is getting bloated.

File: cli/src/rdwatch_cli/__init__.py
Comment (mvandenburgh) [inline cli/src/rdwatch_cli/__init__.py:192]: Why do we no longer need an exception here?

**Review by mvandenburgh — COMMENTED** [review]
This works correctly for me, after running it on the commands you shared on slack :+1:. Just left a couple comments on the code itself.

File: django/src/rdwatch/views/tile.py
Comment (BryonLewis) [reply django/src/rdwatch/views/tile.py:258]: I don't know if we can convince maplibre to use query parameters instead of paths when generating URLS.  If we do it may be a non-standard thing that people may find awkward as you mentioned.  I'd have to look into that to see if it is possible.  I do agree with refactoring the commonality between the endpoints if possible.

File: cli/src/rdwatch_cli/__init__.py
Comment (BryonLewis) [reply cli/src/rdwatch_cli/__init__.py:192]: Without it, the tile version will crash when creating a movie if a single image for a specific x/y/z location isn't found.  I'd much rather have it log and skip the missing the image (maybe I add in the timestamp/bbox) then not create a movie.

Comment (BryonLewis) [reply cli/src/rdwatch_cli/__init__.py:192]: Also, I'm not a fan of the current vocab of the endpoints.  'visual' means worldview is quite a leap, but I didn't want to alter the existing stuff so I used the same pattern.

File: django/src/rdwatch/views/tile.py
Comment (BryonLewis) [reply django/src/rdwatch/views/tile.py:258]: Also, I'm not a fan of the current vocab of the endpoints.  'visual' means worldview is quite a leap, but I didn't want to alter the existing stuff so I used the same pattern.

**Comment by BryonLewis** [comment]
I refactored the requests for bbox/tiles for S2/L8 and worldview to utilize the same function.  Let me know if you think there should be an alternative way to do this.

File: django/src/rdwatch/views/tile.py
Comment (mvandenburgh) [reply django/src/rdwatch/views/tile.py]: Agreed on the endpoint names, I think that's worth revisting once all of the demo code is reviewed and merged.

File: cli/src/rdwatch_cli/__init__.py
Comment (mvandenburgh) [reply cli/src/rdwatch_cli/__init__.py]: Ah I see, yeah I agree that it's better to skip the missing image instead of crashing. In that case, can we use `click.echo(...)` here instead of print?

**Review by mvandenburgh — APPROVED** [review]
LGTM once the dict key issue is fixed :+1:

**Review by mvandenburgh — APPROVED** [review]

**Pull request merged**
